src/app/base-components/base-components.module.ts

```diff
@@ -6,11 +6,12 @@ import { BaseComponentsComponent } from './base-components.component';
 import {ButtonModule} from "../../core/components/buttons/button/button.module";
 import {ButtonOutlineModule} from "../../core/components/buttons/button-outline/button-outline.module";
 import {ButtonTextModule} from "../../core/components/buttons/button-text/button-text.module";
-
+import { ProgressbarComponent } from 'src/core/components/progressbar/progressbar.component';
 
 @NgModule({
   declarations: [
-    BaseComponentsComponent
+    BaseComponentsComponent,
+    ProgressbarComponent
```

Comment:
Das Modul ist zum Deklarieren der Komponenten da, sondern um sie zu importieren und beispielhaft auf einer Demo Seite zu zeigen 😄 
Auf der Base-Components-Seite sollen (wenn man unter /design schaut) die Komponenten einmal in den Variationen gezeigt werden, damit es mit der Design Vorlage abgeglichen werden kann. 
Mit dieser Seite ist dann auch das Review vom Design möglich.